dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunctionLogMessages.cs

```diff
@@ -17,18 +18,38 @@ internal static partial class KernelFunctionLogMessages
     /// </summary>
     [LoggerMessage(
         EventId = 0,
-        Level = LogLevel.Trace, // Sensitive data, logging as trace, disabled by default
-        Message = "Function {FunctionName} invoking. Arguments: {Arguments}.")]
-    public static partial void LogFunctionInvokingWithArguments(
+        Level = LogLevel.Information,
+        Message = "Function {FunctionName} invoking.")]
+    public static partial void LogFunctionInvoking(
         this ILogger logger,
-        string functionName,
-        KernelArguments arguments);
+        string functionName);
+
+    /// <summary>
+    /// Logs arguments to a <see cref="KernelFunction"/>.
+    /// The action provides the benefit of caching the template parsing result for better performance.
+    /// And the public method is a helper to serialize the arguments.
+    /// </summary>
+    private static readonly Action<ILogger, string, Exception?> s_logFunctionArguments =
+        LoggerMessage.Define<string>(
+            logLevel: LogLevel.Trace,   // Sensitive data, logging as trace, disabled by default
+            eventId: 1,
```

Comment:
Need more details on what is `1`, enumerable here may help

Reply:
Same as above